Makefile

```diff
@@ -98,7 +98,6 @@ open-proxy:
 DB_DATA_DIR ?= container-data/db
 DB_CONTAINER_NAME ?= prod-kscout-bot-api-db
 DB_USER ?= prod-kscout-bot-api
-DB_PASSWORD ?= secretpassword
```

Comment:
Why was `DB_PASSWORD` removed? It is still used in the Makefile? 

```
DB_PASSWORD ?= secretpassword
```
Makes it so the `DB_PASSWORD` variable will be set to `secretpassword` by default.

Also can we move these variables to the top of the Makefile where the other variables are?

Reply:
Was thinking of keeping password only ENV and not visible in Makefile

Reply:
Why? This will just make development harder for people. Because they will have to manually set this variable.

The variable is set in the Makefile isn't used in production. It is used only for local development. Setting a default value for `DB_PASSWORD` makes it so most people can just type `make db` and `make db-cli` with little effort.

- [ ] Add `DB_PASSWORD` variable back with default value

Reply:
#GetTheDBPasswordBack

Reply:
:joy:

Reply:
DB Password Gate

Reply:
😂